homework.cls

```diff
@@ -81,6 +91,15 @@
 % Page formatting %
 %%%%%%%%%%%%%%%%%%%
 
+
+% make the margins a bit smaller
+\RequirePackage[
+  paper=a4paper,
+  headheight=14pt,
+  margin=1in,
+  marginparwidth=70pt
+]{geometry}
```

Comment:
This should be left to the user in my opinion. I personally prefer the default large margins, since I think they help readability (cf. https://tex.stackexchange.com/a/71211/23505)

Reply:
Again this was my personal preference, but thanks for that resource. It's really interesting. Making me reconsider smaller margins 🤔